+Base/GitPanel.m

```diff
@@ -0,0 +1,164 @@
+classdef GitPanel
+    
+    properties
+        panel;      % uipanel
+        control;    % uicontrol that displays the current branch info. % For some reason, this property is empty from inside the object, but not from outside. Using panel.Children to access this uicontrol.
+        menu;       % uicontextmenu to show useful git options.
+    end
+    
+    properties (Constant)
+        enableLeftClick = false
+    end
+    
+    methods
+        function obj = GitPanel(varargin)
+            if numel(varargin) == 2
```

Comment:
I would encourage updating this to use MATLAB's [inputParser](https://www.mathworks.com/help/matlab/ref/inputparser.html).

Reply:
Not worth it in this case, as only two situations are supported (in CC, and debug). I don't think it's worth adding more situations because they shouldn't be used.